tests/integration_tests/util.py

```diff
@@ -52,6 +52,15 @@ def verify_clean_log(log):
             'http://169.254.169.254/latest/meta-data/')
         warning_texts.append(fetch_error_text)
         traceback_texts.append(fetch_error_text)
+        # Oracle has a file in /etc/cloud/cloud.cfg.d that contains
+        # users:
+        # - default
+        # - name: opc
+        #   ssh_redirect_user: true
+        # This can trigger a warning about opc having no public key
+        warning_texts.append(
```

Comment:
Not a big deal for this integration test, but seems like we might want instance.cloud awareness in verify_clean_logs for azure vs oracle cases. But, this can wait until we have a third use-case.